src/components/dockerRegistry/Docker.tsx

```diff
@@ -1513,14 +1517,19 @@ function DockerForm({
                         <hr className="mt-0 mb-0" />
                     </div>
                 )}
-                <ValidateForm
-                    id={id}
-                    onClickValidate={onClickValidate}
-                    validationError={validationError}
-                    isChartRepo={true}
-                    validationStatus={validationStatus}
-                    configName="registry"
-                />
+                {!isGCROrGCP ||
```

Comment:
It should have AND operations instead of OR

Reply:
Done